greedy.py

```diff
@@ -14,327 +23,278 @@
+                                        for u in range(0,len(dummy_one)-1):
+                                                if dummy_one[u]!=0:
+                                                        flag1 = 1
+                                                        break
+                                        if flag1 == 0:
+                                                for n in range(1,len(dummy_two)):
+                                                        dummy_two[0] += dummy_two[n]
+                                                        if dummy_two[n]!=0:
+                                                                dummy_two[n]=0
+                                                        
+                                        if flagger == 1:
+                                                if (max_eval <= dummy_one[len_one] - dummy_two[0]):
+                                                        max_eval = dummy_one[len_one] - dummy_two[0]
+                                                        final_one = list(dummy_one)
+                                                        final_two = list(dummy_two)
+                                        else:  
+                                                if (max_eval < dummy_one[len_one] - dummy_two[0]):
+                                                        max_eval = dummy_one[len_one] - dummy_two[0]
+                                                        final_one = list(dummy_one)
+                                                        final_two = list(dummy_two)
+                                        for y in final_two:
+                                                print y,
+                                        print
 
-                                    
-                                                            
+                                        for x in final_one:
+                                                print x,
 
+                                        print
+                                        print 'Last start from is : ',start_from
+                                        print 'Last picked value',picked_value                                                                             
+                             
     
-def main():
-    global task
-    global player_no
-    global cut_off_depth
-    global board_two
-    global board_one
-    global mancala_two
-    global mancala_one
-    global final_one
-    global final_two
-    global flagger
+                dummy_one = list(board_one)
+                dummy_two = list(board_two)
+                for y in dummy_two:
+                        print y,
+                print
+                for x in dummy_one:
+                        print x,
+                print
+                #start = 0
+                                                        
     
-    #InputFileName = str(sys.argv[2])
-    InputFileName = "input_1.txt"
```

Comment:
This might be a little confusing, since the input file is not csv, but any file with well defined delimiters, this is good enough.